src/utils/details.js

```diff
@@ -27,6 +28,15 @@ function _createRequest ({
   ])
 }
 
+function validateHeaders (headers) {
+  for (const name in headers) {
+    const value = headers[name].toString()
+    if (value && value.indexOf('\n') !== -1) {
```

Comment:
I think we need to allow unicode characters so that you can do things like send the sender's name (something ILP Kit is currently doing) in a header with non-ASCII characters.